tests/integration/multiplayer/presence.html

```diff
@@ -0,0 +1,95 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title> Presence Api Integration Test</title>
+    <script src="../../node_modules/jquery/dist/jquery.min.js"></script>
+    <script src="../../../dist/epicenter-multiplayer-dependencies.min.js"></script>
+    <script src="../../../dist/epicenter-edge.js"></script>
+
+	<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+    <style>
+        section {
+            padding: 30px 0;
+            border-top: 1px dotted silver;
+        }
+        .instructions {
+            color: blue;
+        }
```

Comment:
I'd roll this test up into the other integration test, just as a section on the bottom.